code/__HELPERS/_string_lists.dm

```diff
@@ -13,8 +13,8 @@ GLOBAL_VAR(string_filename_current_key)
 
 	if((filepath in GLOB.string_cache) && (key in GLOB.string_cache[filepath]))
 		var/response = pick(GLOB.string_cache[filepath][key])
-		var/regex/r = regex("@pick\\((\\D+?)\\)", "g")
-		response = r.Replace(response, GLOBAL_PROC_REF(strings_subkey_lookup))
+		var/regex/regex_expression = regex("@pick\\((\\D+?)\\)", "g")
+		response = regex_expression.Replace(response, GLOBAL_PROC_REF(strings_subkey_lookup))
```

Comment:
`regex_expression` is no better than `r`

Reply:
Any var name that is an actual word is better, single character var names are so bad partly because it's near impossible to scan through code using them due to it being difficult to differentiate the var names at a glance. Once it's an actual word though that problem is entirely gone, even if you have to check once what a var name is actually being used for.

Reply:
Here, I'll give it a better name.
```suggestion
		var/regex/needs_replacing = regex("@pick\\((\\D+?)\\)", "g")
		response = needs_replacing.Replace(response, GLOBAL_PROC_REF(strings_subkey_lookup))
```